app/services/stats/public/taking_care_time_stats.rb

```diff
@@ -4,8 +4,8 @@ class TakingCareTimeStats
 
     def main_query
       Solicitation
-        .status_processed
-        .joins(diagnosis: [needs: :matches])
+        .joins(:diagnosis, diagnosis: [needs: :matches])
```

Comment:
```suggestion
        .joins(diagnosis: [needs: :matches])
```
C'est équivalent il me semble